modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -202,6 +206,7 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
       if (this._settings.opaOptions.enforced) {
         this.form.get(Controls.OPAIntegration).disable();
       }
+      this.form.get(Controls.KubernetesDashboardEnabled).setValue(this._settings.enableDashboard);
```

Comment:
I think we shouldn't enable it by default (if its enabled in admin settings). Wdyt?

Reply:
@ahmadhamzh please bring back this change but now add an `if` condition here so that we only set this value when dashboard is disabled in admin settings.